src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -0,0 +1,46 @@
+<template >
+    <template if:false={isActiveInstance}>
+        <lightning-card>
+            <div class="spinner" >
```

Comment:
Why wrap the spinner in a div with a class of spinner? I don't see a custom CSS style for spinner? Does this do something else?